drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -2940,12 +2950,18 @@ void RasterizerSceneGLES3::_render_list_template(RenderListParameters *p_params,
 
 			// Can be index count or vertex count
 			uint32_t count = 0;
+
```

Comment:
```suggestion
```
This breaks the association with the definition of the value, and unrelated to your code, please restore